src/Select/index.tsx

```diff
@@ -64,21 +107,26 @@ const ListItemWrapper: React.FC<ListItemWrapperProps> = ({
   });
 
   return (
-    <ListItem
-      css={{ alignItems: "baseline" }}
-      key={element.props.value || element.props.children}
-      {...downshiftItemProps}
-      selected={downshiftItemProps["aria-selected"] === "true"}
-      startIcon={
-        selectionIndicator === "checkmark" ? (
-          selected ? (
-            <IconCheck css={{ height: "100%", width: "100%" }} />
-          ) : null
-        ) : undefined
-      }
-    >
-      {element.props.children}
-    </ListItem>
+    <ClassNames>
+      {({ css }) => {
+        return renderListItem(
+          {
+            className: css({ alignItems: "baseline" }),
```

Comment:
NBD - am I parsing correctly that classnames on the ListItem that's passed in will be replaced rather than merged?

Reply:
Oops nope I think I parsed wrong, the ListItem isn't passed in, this is passed to the ListItem, and the callsite chooses how to merge classnames?

dandy ✨

Reply:
Exactly right. The client will need to handle merging the class names. That function just needs to know to use `cx` instead of `classnames` if merging with something else from emotion.